core/templates/pages/donate-page/donate-page.module.ts

```diff
@@ -19,37 +19,35 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
+angular.module('oppia', [
+  'headroom', require('angular-cookies'), 'ngTouch', 'ngSanitize',
+  'pascalprecht.translate', 'toastr', 'ui.bootstrap'
+]);
+
 import { Component, NgModule, StaticProvider } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
```

Comment:
I think Component and downgradeComponent are not used anymore.

Reply:
Done!